.github/workflows/ci.yml

```diff
@@ -58,7 +58,7 @@ jobs:
           key: ${{ runner.os }}-golang-${{ hashFiles('**/go.sum') }}
           restore-keys: |
             ${{ runner.os }}-golang-
-      - run: make test
+      - run: go test -race -coverprofile=coverage.out -covermode=atomic -timeout 30s --tags=testing ./...
```

Comment:
The `-race` flag has some awkward failures, e.g., on macOS, see [here](https://github.com/golang/go/issues/61229). Shouldn't we do this full-blown testing on, e.g., Linux and drop it for all others?

Reply:
Yes, the `test` job is currently being executed exclusively on Ubuntu.